core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/LimitedSizeLineConsumerWrapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.core.util;
+
+import java.io.IOException;
+
+/**
+ * Wrapper of {@link LineConsumer} which limits total size of consuming lines.
+ *
```

Comment:
Are you sure about lines?
Maybe characters?

Reply:
As for me, `size of consuming characters` sounds strange so as each character has fixed size which can't be limited.

Reply:
Ok. you may rephrase as you want. But ```size of consuming lines``` actually is not true.

``` size of consuming characters``` - a number, not size.

Reply:
I see. What about next variant:
`Wrapper of {@link LineConsumer} object which is limiting total number of characters which this object can consume`

Reply:
ok